libs/lb/lb-rs/src/service/debug.rs

```diff
@@ -27,6 +36,14 @@ pub struct DebugInfo {
     pub last_panic: String,
 }
 
+#[cfg(target_family = "wasm")]
+impl Lb {
+    pub async fn debug_info(&self, _os_info: String) -> LbResult<String> {
+        Ok("NO LOGS FOUND".to_string())
+    }
+}
+
+#[cfg(not(target_family = "wasm"))]
```

Comment:
I think we could provide a better debug info here, at the very least a more descriptive debug info "no debug info for wasm" wold be a better string. But I wonder if something like workspace should just have a wasm mode without the concept of Lb. Or if you should make an issue to support wasm in a more real way. Maybe a high quality github issue documenting current wasm implementation corners cut would be better.